src/LTOCli/Commands/Leasing.py

```diff
@@ -24,13 +25,64 @@ def func(nameSpace, parser):
         transaction.signWith(handle.getAccount(parser))
         if vars(nameSpace)['unsigned'] is False:
             if vars(nameSpace)['account']:
-                transaction.signWith(handle.getAccountFromOption(parser, nameSpace['account'][0]))
+                transaction.signWith(handle.getAccountFromOption(parser, vars(nameSpace)['account'][0]))
             else:
                 transaction.signWith(handle.getAccount(parser))
             if vars(nameSpace)['no_broadcast'] is False:
                 transaction.broadcastTo(handle.getNode())
         elif vars(nameSpace)['no_broadcast'] is False:
             parser.error(
                 "Use the '--unsigned' option only in combination with the '--no-broadcast' option. Type 'lto lease cancel --help' for more informations ")
+
+
+
+
+
+
+
+
+    elif vars(nameSpace)['subparser-name-lease'] == 'list':  # The lease that I'm giving
+
+        node = PublicNode('http://testnet.lto.network')
+
+        if vars(nameSpace)['account']:
+            address = handle.getAccountFromOption(parser, vars(nameSpace)['account'][0]).address
+        else:
+            address = handle.getAccount(parser).address
+
+
+        def leaseList(address):
+            return node.wrapper(api='/leasing/active/{}'.format(address))
+
+        value = leaseList(address)
+        flag = 0
+        for x in value:
+            if x['sender'] == address:  # outbound
+                print(x['sender'], ':', x['amount'])
+                flag +=1
```

Comment:
Flags are usually boolean, to mark if action X occurred. If you expect the if clause to trigger only once, you can have a `return` after it's printed and terminate the function (and in the case it doesn't match just simple print after it, without if).